packages/state-manager/src/sagas/socket/startConnection/startConnection.saga.ts

```diff
@@ -246,12 +246,6 @@ export function subscribe(socket: Socket) {
           ownerCertificate: payload.network.certificate,
         })
       )
-      emit(
-        communitiesActions.storePeerList({
-          communityId: payload.communityId,
-          peerList: payload.network.peers,
-        })
-      )
```

Comment:
Why are we removing this?

Reply:
You noticed that we do not use peers anymore in the registrar for the owner - it's state-manager part of it, we were passing empty array here.